en.json

```
@@ -292,7 +292,9 @@
                         "show_send": "Always show Send Message button",
                         "show_send_desc": "If disabled, the Send Message button will only appear on mobile.",
                         "show_account_age": "Always show account age next to join messages",
-                        "show_account_age_desc": "If disabled, the account age will only be shown for new users."
+                        "show_account_age_desc": "If disabled, the account age will only be shown for new users.",
+                        "bypass_age_gate": "Always bypass age gate for NSFW channels",
+                        "bypass_age_gate_desc": "If disabled, you will be prompted to confirm your age when entering an NSFW channel."
```

Since this is turned off by default, I think something like this would work better:

```suggestion
                        "bypass_age_gate_desc": "If enabled, you will not be prompted to confirm your age when entering an NSFW channel."
```